### ISSUE TITLE
Audio playback cuts out / is lost after calling

### ISSUE THREAD

**Issue by userj** [ubports/ubuntu-touch#944]
### Illustrations

- Device: Hammerhead
- Channel: 16.04/devel
- Build: 2018-10-29/2

### Steps to reproduce

1. Play a show in Podbird
2. While show is playing make a phone call
3. End call

### Expected behavior

- Podbird would continue playing the show where it left off without interruption

### Actual behavior

-Podbird audio resumes for a few seconds, then it cuts out (but keeps playing?), and then sometimes starts playing again (sometimes it dies completely). As a result you miss part of the audio or it has to be restarted.

This may also happen outside of the podbird app with Music app etc. 
Also may be independent of making a call. Sometimes audio seems to get "tired" or the buffer fills up and has to empty before audio continues playing.

**Comment by userj** [comment]
Sometimes this audio stops playing completely after a while without any apparent influence. 
Podbird then needs restarting.

**Comment by Rom4nB** [comment]
I notice this behaviour on my FP2:

### Illustrations

    Device: FP2
    Channel: 16.04/stable
    Build: OTA-6

### Steps to reproduce

Play a song/title in some app that uses background playing (Podbird, Cloud Music, ...)

### Expected behavior

All songs/titles should play normally, also when the app is not in focus or screen switches off.

### Actual behavior

The first song/title, that was started playing when the app was in focus, plays nicely. With the following title/song the playing has a lot of cuts and breaks when the song is playing (similar like when a CPU is too busy to decode an mp3 (happened sometimes in the 2000s and earlier :D ).
When I switch on the screen and shortly put the Podbird/Cloud Music app in focus, the title will play usually fine until the end - until the next title is started and all starts again.

Maybe this is the same bug as you experience with playback issues when a call comes in?

**Comment by userj** [comment]
@Rom4nB Yeah it sounds similar. Anything audio related with playback and network disruption. I think the fact that I make a call is not totally related. It's usually that my call is in transition between wifi and data connections. 

You might also notice that the first song playing will be "connected" to the sound settings from the drop down quick settings. Then, when it rolls over to the next track, it's not reflected in that drop down. Instead you have to pull down on the speaker icon and press the play button. After that point it seems that the sound is not disrupted when it goes into the background.
